coderd/database/querier_test.go

```diff
@@ -2164,11 +2171,125 @@ func TestGetProvisionerJobsByIDsWithQueuePosition(t *testing.T) {
 		t.SkipNow()
 	}
 
+	now := dbtime.Now()
+	ctx := testutil.Context(t, testutil.WaitShort)
+
+	testCases := []struct {
+		name           string
+		jobTags        []database.StringMap
+		daemonTags     []database.StringMap
+		queueSizes     []int64
+		queuePositions []int64
+	}{
+		{
+			name: "test-case-1",
+			jobTags: []database.StringMap{
+				{"a": "1", "b": "2"},
+				{"a": "1"},
+				{"a": "1", "c": "3"},
+			},
+			daemonTags: []database.StringMap{
+				{"a": "1", "b": "2"},
+				{"a": "1"},
+			},
+			queueSizes:     []int64{2, 2, 0},
+			queuePositions: []int64{1, 1, 0},
```

Comment:
This is interesting. If there are 0 provisioners for a given job, we return `0/0`?

Reply:
yes, my decision was based on previous behaviour: https://github.com/coder/coder/blob/main/coderd/database/querier_test.go#L2267-L2277
previously if the job isn't pending (for ex. completed), we return `0 / global queue size`, for ex. `0 / 2`

But it's a good question, now we have two different use-cases when we return `0/0`
- job isn't pending -> `0/0`
- job is pending, but doesn't have a provisioner -> `0/0`

Probably we need to differentiate them, maybe return the `status` along with `queue_pos` & `queue_size`

Reply:
We decided to keep `0/0` for now, and change it later if it will be necessary.